### DISCUSSION TITLE
[Unstable] eye of europa structure collission problems (only in some parts)

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#15058] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://cdn.discordapp.com/attachments/668819422919524352/1303078384829468704/peak.mp4?ex=672a71d0&is=67292050&hm=7ee4d71f224d2d55a18fb98b742235a6e094f8d79fceefed1967f1e12bb97907&
in the first level of eye of europa the entire structure can be phased through

in the second part the door that leads to "the exit?" will still have collission

### Reproduction steps
play on the eye of europa

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Seems to be a symptom of the same bug as https://github.com/FakeFishGames/Barotrauma/discussions/15069, a fix should be coming up in the next unstable build.
